src/main/resources/org/jenkins/plugins/lockableresources/LockStep/config.jelly

```diff
@@ -0,0 +1,9 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
+	xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+	<f:entry title="${%Resource}" field="resource">
+		<f:textbox/>
+	</f:entry>
+	<f:entry title="${%Inverse precedence}" field="inversePrecedence">
```

Comment:
I wonder if this should be checked by default. It seems like the behavior you would normally want.